runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
Correct me if I'm wrong, I think the current hooks today run after the process exits, so it's already the case today that you can't use a hook to do this. If you want a container that exists to be a sandbox (I.e. It's namespaces outlive a particular process) why not just create one container to be a sandbox and then another that enters its namespaces (but not its pid namespace). You can then use a post-stop hook on the sub-container (whose death will not affect the mount namespace of the sandbox container) to do whatever you want?